std_msgs/Header header
    
uint16 u_status_sensor
uint32 u_id
uint16 u_age
uint8 u_status_measurement
uint8 u_status_movement
uint16 u_position_invalid_flags
uint8 u_position_reference
float32 u_position_x
float32 u_position_y
float32 u_position_z
float32 u_position_x_std
float32 u_position_y_std
float32 u_position_z_std
float32 u_position_covariance_xy
float32 u_position_orientation
float32 u_position_orientation_std
uint8 u_existence_invalid_flags
float32 u_existence_probability
float32 u_existence_ppv
uint8 u_classification_car
uint8 u_classification_truck
uint8 u_classification_motorcycle
uint8 u_classification_bicycle
uint8 u_classification_pedestrian
uint8 u_classification_animal
uint8 u_classification_hazard
uint8 u_classification_unknown
uint8 u_classification_overdrivable
uint8 u_classification_underdrivable
uint8 u_dynamics_abs_vel_invalid_flags
float32 f_dynamics_abs_vel_x
float32 f_dynamics_abs_vel_y
float32 f_dynamics_abs_vel_x_std
float32 f_dynamics_abs_vel_y_std
float32 f_dynamics_abs_vel_covariance_xy
uint8 u_dynamics_rel_vel_invalid_flags
float32 f_dynamics_rel_vel_x
float32 f_dynamics_rel_vel_y
float32 f_dynamics_rel_vel_x_std
float32 f_dynamics_rel_vel_y_std
float32 f_dynamics_rel_vel_covariance_xy
uint8 u_dynamics_abs_accel_invalid_flags
float32 f_dynamics_abs_accel_x
float32 f_dynamics_abs_accel_y
float32 f_dynamics_abs_accel_x_std
float32 f_dynamics_abs_accel_y_std
float32 f_dynamics_abs_accel_covariance_xy
uint8 u_dynamics_rel_accel_invalid_flags
float32 f_dynamics_rel_accel_x
float32 f_dynamics_rel_accel_y
float32 f_dynamics_rel_accel_x_std
float32 f_dynamics_rel_accel_y_std
float32 f_dynamics_rel_accel_covariance_xy
uint8 u_dynamics_orientation_invalid_flags
float32 u_dynamics_orientation_rate_mean
float32 u_dynamics_orientation_rate_std
uint32 u_shape_length_status
uint8 u_shape_length_edge_invalid_flags
float32 u_shape_length_edge_mean
float32 u_shape_length_edge_std
uint32 u_shape_width_status
uint8 u_shape_width_edge_invalid_flags
float32 u_shape_width_edge_mean
float32 u_shape_width_edge_std